pom.xml

```diff
@@ -31,9 +31,8 @@
     </distributionManagement>
 
     <properties>
-        <dependency.alfresco-community-repo.version>8.329</dependency.alfresco-community-repo.version>
-        <dependency.alfresco-enterprise-repo.version>8.339</dependency.alfresco-enterprise-repo.version>
-
+        <dependency.alfresco-community-repo.version>8.330-SNAPSHOT</dependency.alfresco-community-repo.version>
```

Comment:
@adinapitul are these ok to be committed with SNAPSHOT?

Reply:
nope, will remove before merging.